### ISSUE TITLE
Can't get webform submission data in views

### ISSUE THREAD

**Issue by combert** [backdrop-contrib/webform#151]
Hi

When i try to create a view of type "Webform submissions" and then try to get a field of type "Webform submission data: Value" it just adds the field, but does not show the fields settings.

After adding the field nothing happens if I try to click the field link in views. There is a short loading gif, and then nothing.

Chrome console gives this error:

POST https://xxxxxxxxx.com/admin/structure/views/ajax/config-item/subs/default/field/value 500 (Internal Server Error)

Has anybody else experienced this?

**Comment by indigoxela** [comment]
Hi @combert,

Just to make sure:
In order to have actual submission values (Component data) available, you need to select a specific "Webform node" first.
That's an autocomplete field. If you start typing, you get suggestions of existing webform nodes.

No, I never had such errors with webform data.

Does your dblog provide more details? /admin/reports/dblog

**Comment by combert** [comment]
The problem is I don't even get to the setting screen where I can select a webform node. 

I just select the checkbox next to Webform submissions: Data and click apply. Then nothing happens. If I the close the modal I can see that the field has been added anyways. But then nothing happens if I click the field. So I cannot access the fields settings.

I emptied the error log and tried to create the view. I still hve the problem, but there is nothing in the error log.

The same thing happens if I try to add "Webform submissions: Data" in Relationships.

**Comment by combert** [comment]
I select the field:
![error1](https://user-images.githubusercontent.com/862681/65040920-d590e080-d955-11e9-9d80-cd944924a965.png)

Efter clicking Add and configure:
![error2](https://user-images.githubusercontent.com/862681/65040927-d88bd100-d955-11e9-95fc-7cd29e3d04f9.png)

After clicking Annuller (Cancel)
![error3](https://user-images.githubusercontent.com/862681/65040934-daee2b00-d955-11e9-8cc7-7d017985271d.png)

After that nothing happens if I click the fields link. Except I get the error in Chrome console listed above.

**Comment by olafgrabienski** [comment]
Once I had a similar problem in Views but not with a Webform field, I don't remember exactly. After saving the View and reopening its configuration (and maybe also refreshing the page), I was able to access the field setting. Want to give it a try?

**Comment by combert** [comment]
Hmm. Tried it. Still the same problem :-(

**Comment by combert** [comment]
So I looked in the error log files and saw this:

[Mon Oct 28 11:18:44.530131 2019] [lsapi:error] [pid 794471] [client xx.xx.xx.xx:61130] [host xxxxx.dk] Backend fatal error: PHP Fatal error:  Call to undefined function views_ui_build_form_url() in /var/www/www.xxxxxxx.dk/xxxxxx/modules/webform/views/webform.views.inc on line 678\n, referer: https://xxxxxxx.dk/admin/structure/views/view/subs

So it seems the function has been renamed in views ui from views_ui_build_form_url to views_ui_build_form_path

I don't know if this is the case? And also why is it only a problem on that one field?

**Comment by herbdool** [comment]
I can't replicate your error @combert 

I'm not sure how you were able to add the views field without selecting a node. Or maybe you did select one but it's not showing up?

I tested on a fresh site. I created a webform with a first and last name. Added some entries. Then created a View with submission data, I chose my webform node then chose First Name. The data shows up fine and no errors.

![Screenshot from 2019-10-29 09-43-20](https://user-images.githubusercontent.com/1071483/67772767-06723400-fa31-11e9-84cd-85b6a1a9feda.png)
![Screenshot from 2019-10-29 09-43-54](https://user-images.githubusercontent.com/1071483/67772768-06723400-fa31-11e9-89ac-294ffbf3e575.png)

But somehow you've triggered a fatal error. I'll look at the renamed function.
